go/main.go

```diff
@@ -16,6 +20,7 @@ import (
 const (
 	accessTokenURL   = "https://api.creditkudos-staging.com/v2/oauth/token"
```

Comment:
i really think these should point to our production env. Seems a bit confusing a client of ours being told to connect to staging